### DISCUSSION TITLE
How do I disable eslint-disable related comments

### DISCUSSION THREAD

**Discussion by laterdayi** [eslint/eslint#16937] [Q&A] [answered]
### Syntax name

*

### Syntax proposal URL

*

### Example code

```js
How do I disable eslint-disable related comments
```

### Implementation Checklist

- [ ] Ecma262 update: 
- [ ] ESTree update: 
- [ ] Acorn update: 
- [ ] `eslint-visitor-keys` update: 
- [ ] `espree` update: 
- [ ] `eslint-scope` update: 
- [ ] `eslint` update: 

### Participation

- [ ] I am willing to submit a pull request for this change.

**Comment by Rec0iL99** [comment]
Hi @laterdayi, if you want to disable ESLint for a part of a file you can use `eslint-disable` in a block comment and then `eslint-enable` in a block comment to enable ESLint again.

```js
/* eslint-disable */

// ESLint will not run for part of the file
const foo = 1

/* eslint-enable */
```

Here's a playground [demo](https://eslint.org/play/#eyJ0ZXh0IjoiLyogZXNsaW50LWRpc2FibGUgKi9cblxuY29uc3QgZm9vID0gMVxuXG4vKiBlc2xpbnQtZW5hYmxlICovXG5cbmNvbnN0IGJhciA9IDIiLCJvcHRpb25zIjp7InBhcnNlck9wdGlvbnMiOnsiZWNtYVZlcnNpb24iOiJsYXRlc3QiLCJzb3VyY2VUeXBlIjoibW9kdWxlIiwiZWNtYUZlYXR1cmVzIjp7fX0sInJ1bGVzIjp7Im5vLXVudXNlZC12YXJzIjpbImVycm9yIl19LCJlbnYiOnsiZXM2Ijp0cnVlfX19)

If you want to disable ESLint for the entire file you can put `eslint-disable` in a block comment at the top of your file.

More options can be found in the [docs](https://eslint.org/docs/latest/use/configure/rules#disabling-rules)

**Comment by nzakas** [comment, marked as answer]
You probably want the [`--no-inline-config`](https://eslint.org/docs/latest/use/command-line-interface#--no-inline-config) CLI option.
